index.html

```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
+<html lang="en">
+  <div id="navbar">
+  <a class="active" href="#home">Home</a>
+  <a href="#news">News</a>
+</div>
```

Comment:
Unnecessary code makes your project look messy and difficult to read. How about you delete this closing div tag? Thank you.